src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1802,58 +1804,85 @@ private void visitEnumType(String symbolName, String qualifiedName, EnumType typ
         return;
       }
 
-      boolean isNumericEnum = type.getElementsType().getPrimitiveType().equals(numberType);
-      if (isNumericEnum) {
-        emit("enum");
+      JSType primitiveType = type.getElementsType().getPrimitiveType();
+      if (maybeStringOrNumericEnum(primitiveType, unqualifiedName, node)) {
+        return;
+      }
+
```

Comment:
add a comment here '// since this is an enum of type different from string literals or numbers, we cannot emit it as TS enum. Instead we emit it as a var/type alias pair'